### PR TITLE
Feature/653/clone

### PR DESCRIPTION
**Pull request by davikstone2**

### CODE REVIEW

File: scripts/bootstrap_linux.sh [hunk 1]
```diff
@@ -10,3 +10,4 @@ cargo install wasm-pack
 cargo build
```
Comment (SankaD) [inline scripts/bootstrap_linux.sh:10]: better to add a step to install cargo/rust and other required tools before this

Comment (SankaD) [reply scripts/bootstrap_linux.sh:10]: better to ask from user before each install though

**Review by SankaD — CHANGES_REQUESTED** [review]

File: scripts/initial_setup.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  Linux)
+    echo "Running bootstrap_linux.sh for Linux"
+    "$SCRIPT_DIR/bootstrap_linux.sh" || { echo "Installation aborted by user. Exiting."; exit 1; }
+    ;;
+  *)
+    echo "Unsupported operating system: $OS"
+    exit 1
+    ;;
+esac
+
```
Comment (SankaD) [inline scripts/initial_setup.sh:74]: - [ ] if a software is already installed, skip to next step without the prompt. \
- [ ] wasm-pack is an optional requirement for wasm compilation. skip that when running initial_setup script

**Comment by arpee** [comment]
What am I testing?

**Comment by arpee** [comment]
Notes:

 - we should not be offering 'connected' nodes to users right now. (These risk bringing down prod).
  (if we do want to keep connected we should maybe offer connection to the _test_ or _staging_ networks.)
 - "isolated node" is confusing "local development node" would be better
 - output paths should be relative so the users knows where things are happening. eg: `config.json has been created from config.template.json.` does not tell the user _where_ these files are...  `./config/config.json` has been created from `./config/config.template.json` is clear.
 - ```blocks folder has been created.
issuance file has been created from issuance.template.
Running bootstrap_linux.sh for Linux
``` same issue. Also what does this script do? 'Installing requirements' or similar helpful so I know what the script is doing. (nice OS detection)
- No need for: ```
chmod +x "$SCRIPT_DIR/bootstrap_mac.sh"
chmod +x "$SCRIPT_DIR/bootstrap_linux.sh"
``` we can just commit the files as executable (I have done this - working on linux).

 - why do I need root to run the node? (nice user permissions detection)
 - `Please answer yes or no.` usually the options are given with capital `... (y/N)` or `...(Y/n)` for the default allowing an enter to move forward and a headless install with all options default to be created later. 
 - Note: `sudo: apt: command not found` we might want to note that this is a deb/ubuntu only script.
 - Is there a reason we don't just add a `./data/blocks/.gitkeep` file in the blocks folder and then just have it empty in the repo?
 - ```# Start node
cd "$BASE_PATH"

echo "Running 'RUST_LOG=debug cargo run' in $BASE_PATH"
env RUST_LOG=debug cargo run
``` we can check if the user want's to start the node, and just give them instructions on how to do that later if they don't want to. (Also we should explain how to end the process).

**Review by SankaD — APPROVED** [review]

**Pull request merged**